### PR TITLE
function for calculating expectation of native contacts 

### PR DESCRIPTION
**Pull request by mrshirts**
Code to calculate expectation of fraction of native contacts.

* example code in examples/evaluating_folding/native_contacts/Q_as_function_of_T.py  Note that the native contact list is somewhat arbitrary at this point (every 3rd sidechain), as is the cutoff.  The example output.nc and output_checklist.nc (coordinates are stored in output_checklist.nc) are not uploaded, should use another one as example.  
* Also added a class method to save a cg_model, with example in examples/storing_cg_model/getandstore_cgmodel.py.  It appears to give a simulation with the same energy , though better tests should be constructed.
* Deleted some old example files.
* Sped up the distance calculation, though it should eventually be converted to numpy for speed (or use something like mdtraj/mdanalysis)
* Adjusted units for kB in some files (need to validate this is correct).

### CODE REVIEW

File: foldamers/parameters/reweight.py [hunk 1]
```diff
@@ -284,9 +284,6 @@ def get_mbar_expectation(E_kln, temperature_list, NumIntermediates, output=None,
     if mbar == None:
         NumTemps = len(temperature_list)  # Last TEMP # + 1 (start counting at 1)
 
-        kB = unit.Quantity(
```
Comment (cwalker7) [inline foldamers/parameters/reweight.py:287]: This gas constant is slightly different than the one stored in simtk units (0.008314472). It looks like their reference values are a little outdated (from 2002 NIST data). https://simtk.org/api_docs/molmodel/api_docs22/Simbody/html/group__PhysConstants.html#_details. They are definitely aware of this: https://github.com/openmm/openmm/issues/2427

File: foldamers/utilities/util.py [hunk 2]
```diff
@@ -203,8 +203,7 @@ def distances(interaction_list, positions):
     distance_list = []
 
     for interaction in interaction_list:
-        if interaction[0] < len(positions) and interaction[1] < len(positions):
-            distance_list.append(distance(positions[interaction[0]], positions[interaction[1]]))
+        distance_list.append(distance(positions[interaction[0]], positions[interaction[1]]))
```
Comment (cwalker7) [inline foldamers/utilities/util.py:207]: It should be fairly easy to replace this with mdtraj.compute_distances. http://mdtraj.org/1.9.3/api/generated/mdtraj.compute_distances.html
We just need to make the pair list into an [n_pair x 2] numpy array. 

I can make that a separate issue though.

**Comment by cwalker7** [comment]
Did you mean to add a new example file Q_as_function_of_T.py? I'm not seeing it anywhere in the repository.

**Comment by mrshirts** [comment]
Indeed.  Added in in #26.  Probably should have waited for someone to review for the merge.  . . .

**Pull request merged**